cluster-install/install.sh

```diff
@@ -73,6 +73,7 @@ readonly __DEFAULT_TRIDENT_ACP_IMAGE_NAME="trident-acp"
 
 readonly __DEFAULT_CONNECTOR_NAMESPACE="astra-connector"
 readonly __DEFAULT_TRIDENT_NAMESPACE="trident"
+readonly __DEFAULT_TORC_NAME="trident"
```

Comment:
The TORC name is used in a few places, this should make things a bit more consistent for the long term.